scripts/test-cov.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 set -e
 
-node_modules/.bin/nyc node_modules/mocha/bin/_mocha --opts test/mocha.opts `scripts/_get-test-directories.sh`
-node_modules/.bin/nyc report --reporter=json
+runSequentially=""
+
+if [ "$CI" ]; then
+  runSequentially="--i"
```

Comment:
You are right. If I may, in addition, I would suggest using `--runInBand` as it might be more self-explanatory than the alias `-i`.